modules/metering-install-operator.adoc

```diff
@@ -12,6 +12,12 @@ You can install metering by deploying the Metering Operator. The Metering Operat
 You cannot create a Project starting with `openshift-` using the web console or by using the `oc new-project` command in the CLI.
 ====
 
+[NOTE]
+====
+If the Metering Operator is installed using a namespace other than `openshift-metering`, the ChargeBack reports are only viewable using the CLI. It is strongly suggested throughout the installation steps to use the `openshift-metering` namespace.
```

Comment:
Chargeback is the old name for Metering so the line should be updated to:
`the Metering reports are only ....`